sql/parser/expr.go

```diff
@@ -233,17 +231,13 @@ func (node *CoalesceExpr) String() string {
 }
 
 // IntVal represents an integer.
-type IntVal int64
+type IntVal struct {
+	Val int64
```

Comment:
I know we pretty much export all the fields on our AST structures, but perhaps we shouldn't?